### PR TITLE
webui: tidy up some test files

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: ui/webui/test/check-storage [hunk 1]
```diff
@@ -377,6 +377,17 @@ class TestUtils():
 
         return True
 
+    def check_mountpoint_table_row(self, b, row, mountpoint=None, device=None, reformat=None, format_type=None):
```
Comment (VladimirSlavik) [inline ui/webui/test/check-storage]: You should be able to drop `b` and use just `self.browser`, I think.

The file is too large to read in github review, sadly, so I apologize if that's actually not the case.

**Review by rvykydal — APPROVED** [review]
Nice cleanup. Thank you.

**Review by VladimirSlavik — COMMENTED** [review]
I wonder how it works, when the `Storage*` classes have `self.browser` on all of them and then they're merged into one descendant. It likely overwrites the variable with the same value and does not matter since it's the same browser. Just... huh.

I also notice that you keep passing the browser everywhere, even if it's an attribute of the class.

**Pull request merged**